### ISSUE TITLE
issue to open zcodedata

### ISSUE THREAD

**Issue by dylanshock** [bonafid3/zcode2gcode#3]
hi,
i try zcode2gcode but i have problem.
I use zsuite the last version setting m200 plus ( create zcodex file).
After unzip with win rar... but the windows application cant open Zcodedata...only .zcode file
wrong some step?

**Comment by ponzifex** [comment]
You are right. There is a filter in the file dialog, but you can just type the path into the editbox field by hand and the application will read the ZCodeData file without extension.
For example "C:\path\to\ZCodeData"
Or just rename the file to ZCodeData.zcode until we push a fix.

**Comment by dylanshock** [comment]
works perfectly thanks friend of your work ... it would be nice to add start gcode and end gcode..without opening repetierhost to add once gcode is created
